nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -42,8 +42,14 @@ In addition to numerous new and upgraded packages, this release has the followin
   }
   ```
 
+- Cinnamon has been updated to 6.0. Please be aware that the [Wayland session](https://blog.linuxmint.com/?p=4591) is still experimental in this release and could potentially [affect Xorg sessions](https://blog.linuxmint.com/?p=4639). We suggest a reboot when switching between sessions.
+
+- GNOME has been updated to 46 "Kathmandu", which introduces an enhanced Files app, support for OneDrive in Online Accounts, and more changes. Refer to the [release notes](https://release.gnome.org/46/) for more details.
```

Comment:
This sounds to me like the Files app is new but in fact it is just minor improvements of the existing app. See also the [release note entry](https://github.com/NixOS/nixpkgs/commit/86cc033bf9a44225c8caadb2e2ec6f02feee02eb) we added in the GNOME 46 PR (not sure why it disappeared).

Reply:
I've updated the Discourse post following this. Will open PRs against nixpkgs and nixos-homepage.